tools/rimage/src/manifest.c

```diff
@@ -483,10 +483,24 @@ static void man_module_fill_reloc(const struct manifest_module *module,
 	man_module->segment[SOF_MAN_SEGMENT_BSS].flags.r.length = 0;
 }
 
+/* Look for a name among module TOML manifests */
+static int man_module_find_cfg(const struct fw_image_manifest_module *modules, const char *name)
+{
+	unsigned int i;
+
+	for (i = 0; i < modules->mod_man_count; i++) {
+		if (!strncmp(name, (const char *)modules->mod_man[i].name, SOF_MAN_MOD_NAME_LEN))
+			return i;
+	}
+
```

Comment:
Btw, do we need to warn that we cant find the TOML here ?

Reply:
@lgirdwood this was done by the caller before, but I've moved it into the function now